### DISCUSSION TITLE
Support for EV4

### DISCUSSION THREAD

**Discussion by TheGreyDiamond** [XilefTech/CharlieOSX#29] [Ideas]
Will there be support for the EV4? I don't think so, but please dicuss.

**Comment by XilefTech** [comment]
I am currently not planning on doing anything for the EV4 - uncreative naming from Lego btw.
The main aim for CharlieOSX is the menu system and that you can control the entire robot without writing actual code and/or having a laptop nearby.
This will be really hard to achieve on that LED-Matrix the ev4 got downgraded to. 

If there would be something like an "ev4 advanced edition" with a better display, I might take some time to (at least in theory) make it work with it. I don't have the money laying around to buy myself another brick that I only need for testing my code so I would rely on the community to help me make it work
